frappe/installer.py

```diff
@@ -301,10 +302,12 @@ def install_app(name, verbose=False, set_as_patched=True, force=False):
 	sync_jobs()
 	sync_fixtures(name)
 	sync_customizations(name)
+	sync_dashboards(name)
 
 	for after_sync in app_hooks.after_sync or []:
 		frappe.get_attr(after_sync)()  #
 
+	frappe.db.commit()
```

Comment:
maybe this should be moved where the command `install-app` is defined (?) 🤔 

This has the potential of partial committing of data in the case of restore, reinstallation - though that would also be the case when someone has exported custom field(s)